corehq/apps/ota/tests/test_claim.py

```diff
@@ -111,6 +112,15 @@ def test_closed_claim(self):
         first_claim = get_first_claim(DOMAIN, self.user.user_id, self.host_case_id)
         self.assertIsNone(first_claim)
 
+    @run_with_all_backends
+    def test_claim_case_other_domain(self):
+        malicious_domain = 'malicious_domain'
+        create_domain(malicious_domain)
```

Comment:
does this need to be cleaned up?

Reply:
Yes, what Simon said. I like to use a context manager for cleaning up, but you could also ...
```suggestion
        dom = create_domain(malicious_domain)
        self.addCleanup(dom.delete)
```

Reply:
Thanks. Updated.